pkg/cmd/roachtest/tests/backup_fixtures.go

```diff
@@ -269,6 +269,56 @@ func registerBackupFixtures(r registry.Registry) {
 			skip:   "only for fixture generation",
 			suites: registry.Suites(registry.Nightly),
 		},
+		{
+			// 400GB backup fixture, no revision history, with 48 incremental layers.
+			// This will used by the online restore roachtests. During 24.2
+			// development, we can use it to enable OR of incremental backups.
+			//
+			// Sadly, because msbutler goofed, the fixture only has 5 incremental
+			// backups. If want to test a long incremental chain in 24.2, just set the
+			// ignorExistingBackups flag to true, and rerun.
+			hardware: makeHardwareSpecs(hardwareSpecs{workloadNode: true}),
+			scheduledBackupSpecs: makeBackupFixtureSpecs(scheduledBackupSpecs{
+				backupSpecs: backupSpecs{
+					version:            fixtureFromMasterVersion,
+					nonRevisionHistory: true,
+				},
+			}),
+			timeout: 5 * time.Hour,
+			initWorkloadViaRestore: &restoreSpecs{
+				backup: backupSpecs{
+					version:           fixtureFromMasterVersion,
+					numBackupsInChain: 48,
```

Comment:
would it be feasible to rename to incCount=5?

Reply:
done!